src/main/java/springboot/webapp/usersmanager/controllers/UserController.java

```diff
@@ -0,0 +1,58 @@
+package springboot.webapp.usersmanager.controllers;
+
+import lombok.AllArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import springboot.webapp.usersmanager.Services.UserService;
+import springboot.webapp.usersmanager.entities.User;
+
+import java.util.List;
+
+@RestController
+@AllArgsConstructor
+@RequestMapping("/api")
+public class UserController {
+
+    private UserService userService;
+
+    @GetMapping("/users")
+    public ResponseEntity<List<User>> getAll() {
+        return new ResponseEntity<>(userService.getAll(), HttpStatus.OK);
```

Comment:
use static methods of ResponseEntity.
for example here ResponseEntity.body(userService.getAll())